definitions/ext-agentless-flex/definition.yml

```diff
@@ -0,0 +1,25 @@
+domain: EXT
+type: AGENTLESS-FLEX
```

Comment:
Could we do EXT-FLEX_REMOTE or EXT-FLEX_AGENTLESS instead (the folder name also needs to change accordingly)? The validator bot is getting lost with the second hyphen.

Reply:
> Could we do EXT-FLEX_REMOTE or EXT-FLEX_AGENTLESS instead (the folder name also needs to change accordingly)? The validator bot is getting lost with the second hyphen.

Thanks for the recommendation I did rename it to **ext-flex_remote**.